cmd/query/app/graphql/graph/schema.resolvers.go

```diff
@@ -0,0 +1,91 @@
+package graph
+
+// This file will be automatically regenerated based on the schema, any resolver implementations
+// will be copied through when generating and any unknown code will be moved to the end.
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/jaegertracing/jaeger/cmd/query/app/graphql/graph/generated"
+	"github.com/jaegertracing/jaeger/cmd/query/app/graphql/graph/model"
+	v11 "github.com/jaegertracing/jaeger/pkg/otel/resource/v1"
+	"github.com/jaegertracing/jaeger/pkg/otel/trace/v1"
+)
+
+func (r *queryResolver) Services(ctx context.Context) ([]string, error) {
+	panic(fmt.Errorf("not implemented"))
+}
+
+func (r *queryResolver) Operations(ctx context.Context, service string) ([]string, error) {
+	panic(fmt.Errorf("not implemented"))
+}
+
+func (r *queryResolver) Traces(ctx context.Context, service string, operationName *string, tags []string, minSpanDuration *int, maxSpanDuration *int, limit *int, startMicros int, endMicros int) ([]*v1.Span, error) {
+	fmt.Println(*limit)
+	fmt.Println(service)
+	fmt.Println(operationName)
+	fmt.Println(tags)
+	return nil, nil
+}
+
+func (r *queryResolver) Trace(ctx context.Context, traceID string) (*model.TracesResponse, error) {
+	preloads := GetPreloads(ctx)
+	names := map[string]bool{}
+	for _, p := range preloads {
+		names[p] = true
+	}
+
+	fmt.Println(traceID)
+	fmt.Println(preloads)
+	s := &v1.Span{
+		TraceId: []byte{0, 1, 2, 3, 4},
+		Name:    "hello",
+	}
+
+	// Remove fields that hasn't been requested
+	// this is a workaround because:
+	// * a custom marshaller is used for OTLP Span
+	//   to ensure compatibility with OTLP JSON (e.g. standard JSON marshalling cannot be used for protos)
+	// * a custom marshaller works well only with scalar types
+	if !names["resourceSpans.instrumentationLibrarySpans.spans.traceId"] {
```

Comment:
Here is the workaround to make selecting object properties/fields work. This is only needed when custom object marshaller is used. The custom marshaller `JSONPb` for `ResourceSpans` has to be used to be compatible with OTLP trace JSON.